content/reference/configuration/agent.md

```diff
@@ -21,6 +21,7 @@ NAME                        | DESC
 `DRONE_PLUGIN_PULL`         | If `true`, plugin updates are pull automatically.
 `DRONE_PLUGIN_NAMESPACE`    | Plugin namespace in dockerhub, defaults to `plugins`.
 `DRONE_PLUGIN_PRIVILEGED`   | Space-separated list of plugins automatically granted privileged mode.
+`DRONE_CONCEAL_SECRETS`     | Conceal secrets from build logs.
```

Comment:
What is the expected value of this env var? true? If so, can you add an "If true" prefix as per some of the other vars?